server/channels/app/session.go

```diff
@@ -115,6 +115,7 @@ func (a *App) GetSession(token string) (*model.Session, *model.AppError) {
 			// gets called from (*WebConn).isMemberOfTeam and revoking a session involves
 			// clearing the webconn cache, which needs the hub again.
 			a.Srv().Go(func() {
+				// TODO(hanzei): Create a clone of c
```

Comment:
Leaving this TODO here for now.

`RevokeSessionById` accepts a `*request.Context`, but `Clone()` returns a `request.CTX`. I didn't want to go down the rabbit hole of changing the pointer to an interface as part of this PR. WIll do that once https://github.com/mattermost/mattermost/pull/24877 is merged.